src/main/check.cpp

```diff
@@ -31,6 +26,8 @@ static size_t hash(const raw_program& raw_prog) {
 }
 
 int main(int argc, char** argv) {
+    ebpf_verifier_options_t ebpf_verifier_options = DEFAULT_EBPF_VERIFIER_OPTIONS;
```

Comment:
I'm probably missing something. Why not initialize with the constant `ebpf_verifier_default_options` and do away with the macro?

Reply:
Good idea, done